actors/builtin/market/testing.go

```diff
@@ -0,0 +1,229 @@
+package market
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/ipfs/go-cid"
+	"github.com/pkg/errors"
+	cbg "github.com/whyrusleeping/cbor-gen"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/specs-actors/v2/actors/builtin"
+	"github.com/filecoin-project/specs-actors/v2/actors/util/adt"
+)
+
+type StateSummary struct {
+	ProposalCount        uint64
+	PendingProposalCount uint64
```

Comment:
I think we'll want more info here for cross-checking with the miner. E.g. all the proposal IDs so we can confirm that the sector deal IDs are a subset. And the pending ones for pre-committed sectors.